Documents/conversions/convert_sqlite.md

```diff
@@ -0,0 +1,145 @@
+# Converting an SQLite Table to other forms
+
+## From SQLite to PostgreSQL
+### Inputs
+* path to sqlite file
+* table name
+* PostgreSQL schema name
+* PostgreSQL table name (defaults to sqlite tablename)
+
+### PostgreSQL prerequisites
+
+The database is called `desc_dc2_drp`.  In order to make changes to the database you must have access to the account `desc_dc2_drp_admin`.  By default PostgreSQL applications like the command-line tool psql will look for account information  in the file `.pgpass` in your home directory, readable (and writable) only by you.  Then you can invoke psql as follows:
+```
+$ psql desc_dc2_drp desc_dc2_drp_admin
+psql (10.15, server 9.6.19)
+Type "help" for help.
+
+desc_dc2_drp=>
+```
+For more information about psql see [https://www.postgresql.org/docs/9.6/app-psql.html](https://www.postgresql.org/docs/9.6/app-psql.html).
+
+### Creating the PostgreSQL schema
+
+From the SQLite table schema make an equivalent .sql file, say called `my_create.sql`, to create the PostgreSQL table. You can see an SQLite schema from within the SQLite command line program by typing
+```
+sqlite> .schema
+```
+Here is the correspondence between SQLite type names and PostgreSQL type names for those types occuring in truth catalogs:
+
+| SQLite         | PostgreSQL       |
+|----------------|------------------|
+| TEXT           | TEXT             |
+| INTEGER or INT | INTEGER          |
+| BIGINT         | BIGINT           |
+| FLOAT          | REAL             |
+| DOUBLE         | DOUBLE PRECISION |
+
+If the original table has ra,dec columns, add a column of type `earth` (a typical name for such a column is `coord` or something containing that string) to the PostgreSQL schema. All of this could perhaps be automated but if so, review output.
```

Comment:
If I want to add a column of type `earth`, do I edit `my_create.sql` with a text editor directly?

Reply:
Yes.  In fact, I'm assuming that, for now, "make an equivalent .sql file" is most likely done with a text editor.  I'll add something about it at that point.